Makefile.am

```diff
@@ -247,6 +247,8 @@ lib_libtarsnap_a_CPPFLAGS=				\
 		-I$(top_srcdir)/lib/network		\
 		-I$(top_srcdir)/lib/scryptenc		\
 		-I$(top_srcdir)/lib/util		\
+		-I$(top_srcdir)/lib-platform		\
+		-I$(top_srcdir)/libarchive		\
```

Comment:
Not sure I understand why this line is here.

Reply:
Oops, mistake.

At one point, I was experimenting with sorting all these lists alphabetically, but then I decided that should be in a separate PR (if I even went through with it).  I evidently didn't finish undoing those changes.